### PR TITLE
973 task update the css class names to match the html controls

### PR DESCRIPTION
**Pull request by bingzhang**
## Description
_Please provide a summary of the pull request and the issue it fixes. Please add necessary details, context, dependencies, explanation of when review is needed (see next section), etc._

**Fixes #(add issue number here and remove parentheses)**

[comment]: # (This project only accepts pull requests related to open issues.)
[comment]: # (If suggesting a new feature or change, please discuss it in an issue first.)
[comment]: # (If fixing a bug, there should be an issue describing it with steps to reproduce.)

## Review Time Estimate
_Please give your idea of how soon this pull request needs to be reviewed by selecting one of the options below. This can be based on the criticality of the issue at hand and/or other relevant factors._

[comment]: # (To select an option, please put an 'x' in the applicable box.)
[comment]: # (If you're unsure about any of these, don't hesitate to ask. We're here to help!.)

- [x] Immediately
- [ ] Within a week
- [ ] When possible

## Type of changes
_Please select a relevant option:_

[comment]: # (To select an option, please put an 'x' in the applicable box.)
[comment]: # (If you're unsure about any of these, don't hesitate to ask. We're here to help!.)

- [ ] Bug fix (non-breaking change which fixes an issue).
- [ ] New feature (non-breaking change which adds functionality).
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected).
- [ ] Other (any another change that does not fall in one of the above categories.)

## Checklist:
_Please select all applicable options:_

[comment]: # (To select your options, please put an 'x' in the all boxes that apply.)
[comment]: # (If you're unsure about any of these, don't hesitate to ask. We're here to help!.)

- [ ] I have signed the Rokwire Contributor License Agreement ([CLA](https://rokwire.org/rokwire_cla)). (_Any contributor who is not an employee of the University of Illinois whose official duties include contributing to the Rokwire software, or who is not paid by the Rokwire project, needs to sign the CLA before their contribution can be accepted._)
- [ ] I have updated the [CHANGELOG](../CHANGELOG.md).
- [ ] I have read the [Contributor Guidelines](../CONTRIBUTING.md).
- [ ] I have performed a self-review of my own code.
- [ ] I have commented my code, particularly in hard-to-understand areas.
- [ ] My change requires updating the documentation.
- [ ] I have made necessary changes to the documentation.
- [ ] I have added tests related to my changes.
- [ ] My changes generate no new warnings.
- [ ] New and existing unit tests pass locally with my changes.
- [ ] Any dependent changes have been merged and published in downstream modules.

---

[comment]: # (Template credit: This pull request template is based on Embedded Artistry {https://github.com/embeddedartistry/templates/blob/master/.github/PULL_REQUEST_TEMPLATE.md}, Clowder {https://github.com/clowder-framework/clowder/blob/develop/.github/PULL_REQUEST_TEMPLATE.md}, and TalAter {https://github.com/TalAter/open-source-templates} templates.)

### CODE REVIEW

File: static/main.css [hunk 1]
```diff
@@ -169,7 +169,7 @@ main.container {
   letter-spacing: normal;
 }
 
-.Events-Manager {
+.hyphenated-words-in-small-caps {
```
Comment (sandeep-ps) [inline static/main.css:172]: Hi, @bingzhang, 

>Let's use `hyphenated-words-in-small-caps` convention for naming CSS classes.

In that message from the previous PR, I was referring to the naming convention and showing it by an example class name and not suggesting using this specific name. Sorry if it wasn't clear. 

Please look at some examples of naming CSS classes. Also, there are other CSS classes in main.css that don't have this naming convention and need to be updated. For example, the `Logout` CSS class name could be changed to `login-logout-button`, indicating the purpose of this class if the same class is being used for both buttons. Please share any further questions here. Thanks.

**Comment by sandeep-ps** [comment]
Also, please include Minu as a reviewer in future Events Manager PRs. Thanks.

File: templates/base.html [hunk 2]
```diff
@@ -155,5 +155,5 @@ <h1 class="Events-Manager">
       <script src="https://cdn.jsdelivr.net/npm/bootstrap-select@1.13.14/dist/js/bootstrap-select.min.js"></script>
     {% endblock %}
   </body>
-<div class="Rectangle-Copy-2"> bottom </div>
+<div class="footer"> bottom </div>
```
Comment (sandeep-ps) [inline templates/base.html:158]: If needed, the text `bottom` could be removed for now and replaced with the appropriate footer content later on. Thanks.

**Review by sandeep-ps — DISMISSED** [review]
The CSS name changes look good. Approving.

File: templates/base.html (hunk 2 above)
Comment (bingzhang) [reply templates/base.html]: removed.

**Comment by bingzhang** [comment]
@sandeep-ps can yo approve again? Thanks

**Review by sandeep-ps — APPROVED** [review]

**Pull request merged**